packages/rum/src/rum.ts

```diff
@@ -257,9 +257,9 @@ function startRumBatch(
         batch.add(withSnakeCaseKeys(event as Context))
       }
     },
-    addUserEvent: (event: RumUserActionEvent) => {
+    addUserEvent: (event: RumUserActionEvent, context?: Context) => {
```

Comment:
What about removing `addUserAction` and add this new parameter to `addRumEvent`?